package.json

```diff
@@ -79,28 +76,32 @@
       "jasmine": true
     }
   },
-  "dependencies": {
+  "peerDependencies": {
+    "carbon-components": "^7.0.0",
     "carbon-icons": "^5.1.2",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2"
+  },
+  "dependencies": {
     "classnames": "2.2.5",
     "lodash.debounce": "^4.0.8",
-    "react": "^15.3.2",
     "react-addons-css-transition-group": "^15.4.2",
-    "react-dom": "^15.3.2",
     "warning": "3.0.0"
   },
   "devDependencies": {
     "@ibma/aat": "1.2.0",
```

Comment:
@ibma/aat? I'm okay with removing it, but we need to get in another tool to scan